api/notd/api/api_v1.py

```diff
@@ -94,6 +95,11 @@ async def update_activity_for_all_collections_deferred():
         await notdManager.update_activity_for_all_collections_deferred()
         return UpdateActivityForAllCollectionsDeferredResponse()
 
+    @router.post('/collection/update-all-token-attributes-deferred', response_model=UpdateAllCollectionTokenAttributesDeferredResponse)
+    async def update_all_collection_token_attributes_deferred():
+        await notdManager.update_all_collection_token_attributes_deferred()
```

Comment:
this should be update_token_attributes_for_all_collections (see the listings and activity example)